Dockerfile-runtime

```diff
@@ -64,6 +64,7 @@ RUN \
   pip install --no-cache-dir -r requirements.txt && \
   pip install --no-cache-dir -r requirements_runtime_demos.txt
 
+RUN pip install --no-cache-dir pytest==3.10.1 pytest-forked==0.2 pytest-xdist==1.22.2 pytest-cov==2.5.1 pylint==1.8.4 pytest-timeout==1.3.3
```

Comment:
In dai we have these requirements in a separated file: `requirements-test.txt` and it is install in docker in a normal way (`pip install -r ...`)

Reply:
Ya, I'd prefer if we only had super basic (pip and setuptools) in Dockerfile, and rest as requirements.